Makefile

```diff
@@ -199,4 +211,31 @@ publish-helm: ## Helm publish
 	helm push ./kro-${RELEASE_VERSION}.tgz oci://${HELM_IMAGE}
 
 .PHONY:
-release: build-image publish-image package-helm publish-helm
\ No newline at end of file
+release: build-image publish-image package-helm publish-helm
+
+##@ Deployment
+
+ifndef ignore-not-found
+  ignore-not-found = true
+endif
+
+.PHONY: install
+install: manifests ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	$(KUBECTL) apply -f ./helm/crds
+
+.PHONY: uninstall
+uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config
+	$(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f ./helm/crds
+
+.PHONY: deploy-kind
+deploy-kind: export KO_DOCKER_REPO=kind.local
+deploy-kind:
+	$(KIND) delete clusters ${KIND_CLUSTER_NAME} || true
+	$(KIND) create cluster --name ${KIND_CLUSTER_NAME}
+	$(KUBECTL) --context kind-$(KIND_CLUSTER_NAME) create namespace kro-system
+	make install
+	# This generates deployment with ko://... used in image. 
+	# ko then intercepts it builds image, pushes to kind node, replaces the image in deployment and applies it
+	helm template kro ./helm --namespace kro-system --set image.pullPolicy=Never --set image.ko=true | $(KO) apply -f -
+	sleep 5
```

Comment:
Is this sleep waiting for the pod to come up?

You can actually do that wait in kubectl instead? Something like this:
`kubectl wait --for=condition=ready --timeout=3m pod -n "$OPERATOR_NAMESPACE" -l control-plane=controller-manager`

You'll need to modify this based on the labels on the KRO pod though.

Reply:
done